### PR TITLE
NOTIF-121 Allow to subscribe per application/bundle

### PR DESCRIPTION
**Pull request by josejulio**
Screenshot_2021-02-22_16-29-30

### CODE REVIEW

**Comment by theute** [comment]
I'm wondering if we need a "All" such as all existing ones but also future event types

**Comment by josejulio** [comment]
> I'm wondering if we need a "All" such as all existing ones but also future event types

Probably could be extended to cover that case. 
Would be there a "All for this bundle" and the global "All"?

File: src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java [hunk 1]
```diff
@@ -212,50 +216,41 @@
...
+                        type
+                ));
     }
 
     @DELETE
-    @Path("/email/subscription/instant")
+    @Path("/email/subscription/{bundleName}/{applicationName}/{type}")
     @RolesAllowed(RbacIdentityProvider.RBAC_WRITE_INTEGRATIONS_ENDPOINTS)
-    public Uni<Boolean> unsubscribeInstantEmail(@Context SecurityContext sec) {
+    public Uni<Boolean> unsubscribeInstantEmail(
```
Comment (pilhuhn) [inline src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java:240]: Nit-pick. As the type is passed as parameter, should the method name changed to no longer contain 'instant' ?
Likewise for subscribe above?

**Review by pilhuhn — APPROVED** [review]
lgtm

**Pull request merged**